omnibus/config/software/datadog-agent.rb

```diff
@@ -8,6 +8,15 @@
 
 name 'datadog-agent'
 
+# STS: Add some dependencies to make this run late in the build, this will avoid early
```

Comment:
are we duplicating those dependencies definitions ?
aren't they somewhere else ?

Reply:
they are duplicate, I was hoping the comment would be clear, let me know how I can improve it